GiniVision/Classes/Core/Screens/Screen API Coordinator/GiniScreenAPICoordinator+Review.swift

```diff
@@ -90,6 +90,9 @@ extension GiniScreenAPICoordinator: MultipageReviewViewControllerDelegate {
     }
     
     @objc fileprivate func closeMultipageScreen() {
+        
+        trackingDelegate?.onReviewScreenEvent(event: Event(type: .back))
```

Comment:
Documentation should mention, that `.back` is triggered only when using the Screen API.